src/base/bittorrent/session.cpp

```diff
@@ -1948,6 +1949,16 @@ bool Session::deleteTorrent(const QString &hash, bool deleteLocalFiles)
 
     delete torrent;
     qDebug("Torrent deleted.");
+    if (isQueueingSystemEnabled()) {
```

Comment:
I would concatenate these conditions with `&&`.

Reply:
ok

Reply:
Done.

Reply:
What's about my other comment?
>I mean to process without any timeout (in the next idle event).

Why do you need additional timeout? Just use 0.

Reply:
Somehow i missed that comment. I thought we're giving it time to delete many torrents but i guess i don't understand how singleshot works. I'll change it to 0

Reply:
done.